### PR TITLE
Update the github plugin with an automatic gh run watch

### PR DESCRIPTION
**Pull request by torbjornvatn**

### CODE REVIEW

**Comment by judoole** [comment]
❤️

**Review by judoole — APPROVED** [review]
Looks very good! Nice!

File: plugins/github/deploy.sh [hunk 1]
```diff
@@ -0,0 +1,21 @@
+#!/usr/bin/env bash
```
Comment (judoole) [inline plugins/github/deploy.sh]: Smart

Comment (torbjornvatn) [reply plugins/github/deploy.sh]: Except that it doesn't work, because it only downloads .mk files when you update brokkr in the project that uses this :/

Comment (torbjornvatn) [reply plugins/github/deploy.sh]: Will look into that

Comment (judoole) [reply plugins/github/deploy.sh]: Hmm... That is true. The Airflow one downloads within the `airflow.mk` https://github.com/unacast/brokkr/blob/master/plugins/airflow/airflow.mk#L94

But it isn't quite nice that either. Maybe if we downloaded the full "folder" instead. Although local paths, AND figuring out every file that is within the folder using curl agains https://github.com is perhaps a challenge.

**Comment by torbjornvatn** [comment]
@judoole I think I solved it!

File: brokkr/brokkr.mk [hunk 2]
```diff
@@ -30,6 +30,9 @@ $(_BROKKR_PLUGINS_MK): $(_BROKKR_PLUGINS_SENTINEL)
 		url="https://raw.githubusercontent.com/$(BROKKR_REPO)/$${plugin_version}/plugins/$${plugin_path}.mk"; \
 		echo "Downloading $${url}"; \
 		curl --fail -s "$$url" -o $(_BROKKR_PLUGINS_DIR)/$${plugin_path}.mk; \
+		url="https://raw.githubusercontent.com/$(BROKKR_REPO)/$${plugin_version}/plugins/$${plugin_path}.sh"; \
+		echo "Downloading $${url}"; \
+		curl --fail -s "$$url" -o $(_BROKKR_PLUGINS_DIR)/$${plugin_path}.sh; \
```
Comment (judoole) [inline brokkr/brokkr.mk:35]: Ha! Smart

Comment (judoole) [reply brokkr/brokkr.mk:35]: I like it

Comment (torbjornvatn) [reply brokkr/brokkr.mk:35]: How did I make a release agian?
Manually?

Comment (judoole) [reply brokkr/brokkr.mk:35]: Yeah, unfortunately no README or fancy routine. Create a release and use the branch, or merge with master and release from that.

**Pull request merged**